testdata/Miseq_PairedEnd/samplesheet.csv

```diff
@@ -0,0 +1,2 @@
+sample,lane,project,fastq_1,fastq_2,rundir
```

Comment:
This is the old format, please check the other data sets and update accordingly:
https://github.com/nf-core/test-datasets/blob/2ce14b964fd7ad84cb6e25c51b202de7052790fa/testdata/MiSeq/samplesheet.csv?plain=1#L1-L2